src/components/VmDetails/cards/UtilizationCard/CpuCharts.js

```diff
@@ -25,8 +25,8 @@ import NoLiveData from './NoLiveData'
  * as a sparkline. Sparkline vales to go from oldest far left to most current on far
  * right.
  */
-const CpuCharts = ({ cpuStats, isRunning, id }) => {
-  const cpuUsed = cpuStats['current.guest'].datum
+const CpuCharts = ({ cpuStats, isRunning, id, sockets, cores }) => {
+  const cpuUsed = cpuStats['current.total'].datum / sockets / cores // the average value considering the number of VM CPUs, same as in Admin Portal
```

Comment:
Have you verified that "current.total" is the VM CPU use from the VM's perspective?

I'm not sure "current.guest" and "current.total" are the same number.

As long as the correct cpu stat is used, the calc to normalize against vCPUs makes sense.

Reply:
I'm not sure what is meant by "verifying rom the VM's perspective".

According to the info I've found, "current.guest" and "current.total" are not the same number:
`cpu.current.total: cpu.current.guest + cpu.current.hypervisor`

And the number displayed in Admin Portal is calculated like this:
`cpu.current.total / ( sockets * cores )`

And as I properly understand this issue, we want to see the same values in the VM Portal, so it seems that we don't need the "current.guest" value but "current.total" instead. Also I've checked that now with this PR, the values in VM Portal are the same as in Admin Portal so the issue should be fixed and we should be consistent in both of the UIs.

Reply:
I don't remember exactly where each number gets collected from.  I believe one number is CPU usage as reported by the guest agent, while another is the CPU usage as reported by vdsm on the VM's host.

I'm not even sure the CPU usage needs to match between webadmin and web-ui.  Webadmin usage is from the admin perspective...how much resources is this VM taking up WRT the host.  Web-ui usage is from the use perspective...how much resources is the OS using WRT the VM.  Subtle difference for sure.

I'll have to look to see if I can find the original discussions about which statistics to use.

Reply:
https://github.com/oVirt/ovirt-engine/blob/a026144f551027629c47a3ccd3f06b57e207dc25/backend/manager/modules/restapi/jaxrs/src/main/java/org/ovirt/engine/api/restapi/resource/VmStatisticalQuery.java#L20-L22

Reply:
Ok, I think you're ok to use the `cpu.current.total`. 

But, change the prop to just `vCPUs` so the chart component only needs to divide by one value.